tests/functional/genesets/e2e_genesets.py

```diff
@@ -0,0 +1,38 @@
+import resdk
+
+from ..base import BaseResdkFunctionalTest
+
+
+class TestCreateGeneset(BaseResdkFunctionalTest):
+    def setUp(self):
+        super().setUp()
+        self.collection = self.res.collection.create(name="Test collection")
+        self.geneset = None
+
+    def tearDown(self):
+        # self.geneset is deleted along with collection
+        self.collection.delete(force=True)
+
+    def test_create_geneset(self):
+        self.geneset = self.res.geneset.create(
+            name="Test name",
+            genes=["FHIT", "MYC"],
+            source="UCSC",
+            species="Homo sapiens",

```

Comment:
I would prefer to chck the "full" version of create, including collection argument.